src/libs/ReportUtils.ts

```diff
@@ -6069,6 +6073,41 @@ function canLeavePolicyExpenseChat(report: OnyxEntry<Report>, policy: OnyxEntry<
     return isPolicyExpenseChat(report) && !(PolicyUtils.isPolicyAdmin(policy) || PolicyUtils.isPolicyOwner(policy, currentUserAccountID ?? -1) || isReportOwner(report));
 }
 
+/**
+ * Whether the user can join a report
+ */
+function canJoinChat(report: OnyxEntry<Report>, parentReportAction: OnyxEntry<ReportAction>, policy: OnyxEntry<Policy>): boolean {
+    if (ReportActionsUtils.isWhisperAction(parentReportAction)) {
```

Comment:
This could use a comment. Is it possible to thread off of a whisper message? 🤔

Reply:
Do you mean that we can disable reply in thread of a whisper message, right?

Reply:
I'm asking why we have this code here. A comment would help explain it. What does it look like to you?

Reply:
Added an explain comment here. And also found, this is added from this PR https://github.com/Expensify/App/pull/31676 that disabling the thread functions for whisper message.

Reply:
Thanks! IMO we don't need to link to the PR in the code. The explanation is enough.